src/modules/core/components/Fields/InputLabel/InputLabel.css

```diff
@@ -38,6 +38,10 @@
   color: var(--dark);
 }
 
+.colorSchemaLightGrey .labelText {
```

Comment:
Please, remove this extra variant and just use the dark colour schema. I think Arren has updated the specs.
Just generally, when you find that there are different colours to our core components, check with Arren please for him to confirm which ones of the existing colours to use.